tools.yml

```diff
@@ -1208,10 +1171,10 @@ tools:
     mem: 100
   toolshed.g2.bx.psu.edu/repos/iuc/chira_map/chira_map/.*:
     cores: 10
-    mem: 80
+    mem: 24
   toolshed.g2.bx.psu.edu/repos/iuc/chira_merge/chira_merge/.*:
     cores: 1
-    mem: 60
```

Comment:
Ok dropping this entirely to just use the default.

Reply:
@mira-miracoli if we don't have this tool in our config, we use the memory from the shared DB isn't it? So we are using 60GB I assume?

@pavanvidem since this is your tool, any recommendation?